packages/seal-loupe-adapter/src/LoupeHelper.php

```diff
@@ -98,6 +98,11 @@ public function reset(): void
         $this->loupe = [];
     }
 
+    public function formatField(string $field): string
```

Comment:
We should mark this method as `@internal`

```suggestion
    /**
     * @internal
     */
    public function formatField(string $field): string
```